internal/infrastructure/event/factory.go

```diff
@@ -22,13 +22,13 @@ func Start(ctx context.Context, config *config.Kafka, handler Eventable) {
 	)
 
 	if consumer, err = NewConsumer(config); err != nil {
-		log.Panic().Msgf("error creating consumer: %s", err.Error())
+		slog.Error("error creating consumer", err.Error())
 		return
 	}
 
 	defer func() {
 		if err := consumer.Close(); err != nil {
-			log.Error().Msgf("error closing consumer: %s", err.Error())
+			slog.Error("error closing consumer", err.Error())
```

Comment:
```suggestion
			slog.Error("error closing consumer", slog.Any("error", err))
```

Reply:
Fixed. Thanks.

Reply:
last state is including this change; resolving this conversation.